### PR TITLE
make the top of the page a bit more compact

### PR DESCRIPTION
**Pull request by neilb**
Hi Slaven,

This is the PR I discussed in email, which just makes the top of the page a bit more compact.

Cheers,
Neil

### CODE REVIEW

File: cgi-bin/cpantestersmatrix.pl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -T
+#!/usr/local/bin/perl -T
```
Comment (eserte) [inline cgi-bin/cpantestersmatrix.pl:1]: The shebang should probably not change...

**Comment by neilb** [comment]
I had changed that for my local perl and forgot to change it back. Reverted now.

**Comment by neilb** [comment]
@eserte nudge

**Comment by eserte** [comment]
Thanks, I also did a deployment right now.

**Pull request merged**